pkgs/development/libraries/mapnik/default.nix

```diff
@@ -1,22 +1,41 @@
-{ lib, stdenv, fetchzip
-, boost, cairo, freetype, gdal, harfbuzz, icu, libjpeg, libpng, libtiff
-, libwebp, libxml2, proj, python3, python ? python3, sqlite, zlib
-, sconsPackages
-
-# supply a postgresql package to enable the PostGIS input plugin
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, pkg-config
+, substituteAll
+, boost
+, cairo
+, freetype
+, gdal
+, harfbuzz
+, icu
+, libjpeg
+, libpng
+, libtiff
+, libwebp
+, libxml2
+, proj
+, python3
+, sqlite
+, zlib
+, catch2
+, scons
+
+  # supply a postgresql package to enable the PostGIS input plugin
 , postgresql ? null
```

Comment:
Please use a boolean to enable this.

Reply:
We can just get rid of the option not to use postgresql.

Reply:
also fine for me